src/EditorFeatures/Core.Wpf/InlineRename/UI/InlineRenameAdornmentManager.cs

```diff
@@ -130,10 +141,19 @@ private void UpdateAdornments()
                 var identifierSelection = new TextSpan(start, length);
 
                 var adornment = new RenameFlyout(
-                    (RenameFlyoutViewModel)s_createdViewModels.GetValue(_renameService.ActiveSession, session => new RenameFlyoutViewModel(session, identifierSelection, registerOleComponent: true, _globalOptionService)),
+                    (RenameFlyoutViewModel)s_createdViewModels.GetValue(
+                        _renameService.ActiveSession,
+                        session => new RenameFlyoutViewModel(session,
+                            identifierSelection,
+                            registerOleComponent: true,
```

Comment:
do you know what the registerOleComponent bit is about?

Reply:
Registering for Ole fails in unit tests (I'd have to dig deeper to remember why) so we have this for testing purposes. Production code should always be true.